README.md

```diff
@@ -63,7 +63,7 @@ Vocabulary is available as
 ### Demo
 
 The showcase of the true extent of Vocabulary can be found in the
-**[interactive storybook](https://cc-vue-vocabulary.netlify.com/storybook)**
+**[interactive storybook](https://cc-vue-vocabulary.netlify.com/)**
```

Comment:
See #180.

Reply:
https://cc-vue-vocabulary.netlify.com/storybook this shows error says page not found however
https://cc-vue-vocabulary.netlify.com/ this takes to the demo of website.